lib/checkMergeConflicts.js

```diff
@@ -10,7 +10,9 @@ module.exports.checkMergeConflicts = async function(context) {
       indexOfPullRequest].number;
     pullRequestDetailsPromiseObj = await context.github.pullRequests.get(
       context.repo({number: pullRequestNumber}));
-
+    // eslint-disable-next-line no-console
```

Comment:
Should this still be here?

Reply:
We have no console log rule in eslintrc enabled. Therefore, we require this. Moreover, I wanted to keep logging to help us in debugging. (Ditto below).